.github/workflows/ci.yml

```diff
@@ -19,5 +19,5 @@ jobs:
     needs: Spec
     uses: "puppetlabs/cat-github-actions/.github/workflows/module_acceptance.yml@main"
     with:
-      flags: "--provision-service"
+      flags: "--provision-service --exclude-platforms '["CentOS-8"]'"
```

Comment:
doesn't it make more sense to remove CentOS 8 from metadata.json?

Reply:
Wanting to keep the supported version on forge. Centos 8 is no longer available from our VM provisioner, and planning to look in to using docker at a later stage

Reply:
CentOS 8 is EoL, Vox Pupuli is removing it from their modules. I think Puppet should do the same. If it's in the metadata.json, people will assume that it's working and tested. That's not the case anymore.